leakcanary-android-sample/src/main/java/com/example/leakcanary/DirectBootReceiver.kt

```diff
@@ -0,0 +1,18 @@
+package com.example.leakcanary
+
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+
+/**
+ * Receiver created during direct boot.
+ *
+ * This allows to verify LeakCanary can properly be installed during direct boot with the sample app,
+ * as it will cause the application to be created when the user hasn't unlocked their device yet.
+ */
+class DirectBootReceiver : BroadcastReceiver() {
```

Comment:
In this a work in progress? The implementation does nothing right now.

Reply:
It is, there is an additional crash in direct boot and haven't gotten the chance to look into it yet. 

The receiver is empty on purpose: it just needs to "exist" so the application class is created in direct boot, thus initializing leak canary

Reply:
Interesting!

> The receiver is empty on purpose: it just needs to "exist" so the application class is created in direct boot, thus initializing leak canary

Worth adding as comment inside the empty method?